05_deploy_bootstrap_vm.sh

```diff
@@ -117,15 +117,13 @@ sudo systemctl reload NetworkManager
 cat /tmp/dnsmasq.conf | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=dnsmasq.conf
 
 # Build and start the ironic container
-cat ironic/runironic.sh | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=runironic.sh
-cat ironic/Dockerfile | ssh -o "StrictHostKeyChecking=no" core@$IP sudo dd of=Dockerfile
-ssh -o "StrictHostKeyChecking=no" core@$IP sudo podman build \
-    --build-arg RHCOS_IMAGE_URL=${RHCOS_IMAGE_URL} \
-    --build-arg RHCOS_IMAGE_VERSION=${RHCOS_IMAGE_VERSION} \
-    --build-arg RHCOS_IMAGE_FILENAME_OPENSTACK=${RHCOS_IMAGE_FILENAME_OPENSTACK} \
-    -t ironic:latest .
+cat "${RHCOS_IMAGE_FILENAME_OPENSTACK}" | ssh -o "StrictHostKeyChecking=no" "core@$IP" sudo dd of="${RHCOS_IMAGE_FILENAME_OPENSTACK}"
+ssh -o "StrictHostKeyChecking=no" "core@$IP" sudo podman pull docker.io/vrutkovs/metalkube-ironic
```

Comment:
Should this be quay.io not docker.io ?

Reply:
Good catch, should come from https://quay.io/repository/metalkube/metalkube-ironic/

Reply:
Fixed